src/components/entity/diff/row.vue

```diff
@@ -11,9 +11,9 @@ except according to the terms contained in the LICENSE file.
 -->
 <template>
   <tr class="entity-diff-row">
-    <td :class="{ property: name !== 'label', conflicting }" v-tooltip.text>
-      {{ name === 'label' ? $t('entity.label') : name }}
-      <span v-if="conflicting" class="sr-only">{{ $t('conflictingProp') }}</span>
+    <td :class="{ property: name !== 'label', conflicting }">
+      <span v-tooltip.text>{{ name === 'label' ? $t('entity.label') : name }}</span>
+      <span v-if="conflicting" class="sr-only">&nbsp;{{ $t('conflictingProp') }}</span>
```

Comment:
With the introduction of the `<span>`, there was no whitespace between the property name and the text of the `.sr-only` element. I think whitespace is needed for screen readers, which is why I added an `&nbsp;`.